.github/workflows/release.yml

```diff
@@ -1,36 +0,0 @@
-name: release
```

Comment:
This file shouldn't be deleted - this is for the release job (that automatically creates Helm releases on **push** to main, not on **PR** to main).

Arguably we should change this to target main branch only, for the same argument that the version-increment job is targetting main only (AIUI there's no protection against a preview branch overwriting a main branch release if we don't change this).